src/components/Settings/helper.js

```diff
@@ -1,3 +1,12 @@
-export const isFetching = requests => {
+import CozyClient from 'cozy-client'
+import compare from 'semver-compare'
+
+export const isFetchingQueries = requests => {
```

Comment:
Nit: I think it would be better / more flexible/composable to extract the

request.fetchStatus === 'loading' into isCollectionLoading 

and then to call directly `requests.some(isCollectionLoading)`

I think it is clearer when reading.

Reply:
Nice idea, I think that this kind of utilities functions should be served by cozyclient itself, wdyt?

Reply:
Yes, it's been a long time that we have isCollectionLoading in Banks, PR incoming.

Reply:
Thanks 😊

Reply:
https://github.com/cozy/cozy-client/pull/576